db/migrate/20191007100156_add_reminder_consent_to_patients.rb

```diff
@@ -0,0 +1,5 @@
+class AddReminderConsentToPatients < ActiveRecord::Migration[5.1]
+  def change
+    add_column :patients, :reminder_consent, :string, index: true, null: false, default: Patient.reminder_consents[:denied]
```

Comment:
Just test this on 140k patients and see if the default column value is updated throughout the table in a reasonable time.